themes/V3/5ePHB/style.less

```diff
@@ -942,8 +942,8 @@ h5 + table{
 			font-family: Overpass;
 			font-size: 0.45cm;
 			position: relative;
-			margin-top: -0.7em;
-			line-height: 1.1em;
+			margin-top: -0.315cm;
+			line-height: 0.495cm;
```

Comment:
And finally, this one is all good

<img width="384" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/58999374/c31c5926-0600-496a-8bf1-9b38e7f9684d">